README.md

```diff
@@ -10,81 +10,84 @@ This plugin allows user to define allow list for ingress and egress packet filte
 
 ## Usage
 
-By default, all ingress and egress traffic is dropped.
-To allow specific ingress and egress CIDR blocks, create a NetworkAttachmentDefinition. See the following example:
+When using the cidr-filtering-cni plugin, all ingress and egress traffic is dropped by default.
+To allow specific ingress and egress CIDR blocks/ports, create configMaps(s) referencing a NetworkAttachmentDefinition by having a label the NetworkAttachmentDefinition spec name.
+Each configMap additionally needs to have "cidr-filtering-cni" label.
```

Comment:
Please add an example to the PR showing how to drop all ingress and egress traffic.

Reply:
Done